src/components/ImageAnimation.vue

```diff
@@ -1,15 +1,34 @@
 <template>
   <div id="template-image-animation">
-
   <div class="image-animation">
 
     <div class="row">
       <div class="col-sm-12" >
         <ul class="nav nav-tabs">
-          <li role="presentation" class="nav-item" :class="{'active': currentAnimationKey === 'delta_fringe_images'}" @click.stop="switchAnimation('delta_fringe_images')"><a href="#">Delta fringe</a></li>
-          <li role="presentation" class="nav-item" :class="{'active': currentAnimationKey === 'channel_network_images'}" @click.stop="switchAnimation('channel_network_images')"><a href="#">Channel network</a></li>
-          <li role="presentation" class="nav-item" :class="{'active': currentAnimationKey === 'sediment_fraction_images'}" @click.stop="switchAnimation('sediment_fraction_images')"><a href="#">Sediment fraction</a></li>
-          <li role="presentation" class="nav-item" :class="{'active': currentAnimationKey === 'subenvironment_images'}" @click.stop="switchAnimation('subenvironment_images')"><a href="#">Sub-environment</a></li>
+          <li role="presentation"
+            class="nav-item"
+            :class="{'active': currentAnimationKey === 'delta_fringe_images'}"
+            @click="switchAnimation('delta_fringe_images')">
+            <a class="nav-link active" href="#">Delta fringe</a>
+          </li>
+          <li role="presentation"
+            class="nav-item"
+            :class="{'active': currentAnimationKey === 'channel_network_images'}"
+            @click="switchAnimation('channel_network_images')">
```

Comment:
switchTo? show? use a  "current" animation?

Reply:
first setup done, need to improve next sprint -> info needs to come from backend